s3cortxutils/s3confstore/s3confstore/s3confstore

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/python3.6
+
```

Comment:
pls add copyright statements for this file

Reply:
@pratyush-seagate do the needful.

Reply:
Sure, will address it in the followup PR.